docs/framework/guides/development-tools.md

```diff
@@ -35,6 +35,17 @@ or as a plain `<script>` tag in the HTML of your application:
 <script src="../node_modules/@ckeditor/ckeditor5-inspector/build/inspector.js"></script>
 ```
 
+#### Inspector as a bookmarklet
+
+Alternatively, you might create a bookmarklet in your browser that will allow you to open inspector on every page without interference
```

Comment:
Some "markdown-codestyle" things:

* Don't wrap the lines manually. Configure your IDE to wrap them for you.
* Replace `<b>` with `**`.
* Add extra padding (empty new lines) between blocks of content. In this case – before and after the code block.

Reply:
Done.